### PR TITLE
multi-region: conditional bootstrap

### PR DESCRIPTION
**Pull request by bernadinm**
This adds control to enable a bootstrap node on the remote region

### CODE REVIEW

File: main.tf
Comment (fatz) [inline main.tf]: needs to be changed before merge

File: main.tf
Comment (fatz) [inline main.tf]: needs to be changed before merge

File: main.tf
Comment (fatz) [inline main.tf]: needs to be changed before merge

File: .gitignore
Comment (fatz) [inline .gitignore]: can we please keep the .gitignore clean. A lot of entries which are not needed for this project

**Review by fatz — CHANGES_REQUESTED** [review]
have to review other modules first. But keeping this as changes requested because of non registry source

**Comment by fatz** [comment]
@bernadinm can you take care of the conflicts?

**Comment by bernadinm** [comment]
complete, thanks @fatz !

**Comment by fatz** [comment]
terraform fmt failed. @bernadinm please have a look at jenkins

**Comment by bernadinm** [comment]
This is blocked by this PR here: https://github.com/dcos-terraform/terraform-aws-bootstrap/pull/6

**Comment by bernadinm** [comment]
conflicts are taken care of, merging

**Pull request merged**
